static/scripts/login.js

```diff
@@ -88,6 +95,14 @@ $(document).ready(() => {
 		$systems.trigger('chosen:updated');
 	};
 
+	const handleSystemChange = (system) => {
+		if (system && system === 'moodle') {
```

Comment:
why show the button for moodle? here, system might have a prop which tells us oauth is enabled for it. using "moodle" seems not to be a good hypothesis to me.

Reply:
it should be iserv, just didnt have the time yet to add a moodle school to the mock database